tests/base.py

```diff
@@ -0,0 +1,196 @@
+        return {
+            'client_secret': os.getenv('TAP_EXACTTARGET_CLIENT_SECRET')
+        }
+
+    def get_properties(self, original: bool = True):
+        return_value = {
+            'start_date': '2014-01-01T00:00:00Z',
+            'client_id': os.getenv('TAP_EXACTTARGET_CLIENT_ID'),
+            'tenant_subdomain': os.getenv('TAP_EXACTTARGET_TENANT_SUBDOMAIN')
+        }
+        if original:
+            return return_value
+
+        # Reassign start date
+        return_value["start_date"] = self.START_DATE
+        return return_value
+
+    def expected_metadata(self):
+        return {
+            "campaign": {
+                self.PRIMARY_KEYS: {"id"},
+                self.REPLICATION_METHOD: self.FULL_TABLE
+            },
+            "content_area":{
+                self.PRIMARY_KEYS: {"ID"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"ModifiedDate"},
+            },
+            "data_extension.test emails":{
+                self.PRIMARY_KEYS: {"_CustomObjectKey", "ID"},
+                self.REPLICATION_METHOD: self.FULL_TABLE,
+            },
+            "data_extension.This is a test":{
+                self.PRIMARY_KEYS: {"_CustomObjectKey", "ID"},
+                self.REPLICATION_METHOD: self.FULL_TABLE,
+            },
+            "data_extension.my_test":{
+                self.PRIMARY_KEYS: {"_CustomObjectKey", "ID"},
+                self.REPLICATION_METHOD: self.FULL_TABLE,
+            },
+            "data_extension.test 1":{
+                self.PRIMARY_KEYS: {"_CustomObjectKey", "ID"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"JoinDate"},
+            },
+            "email":{
+                self.PRIMARY_KEYS: {"ID"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"ModifiedDate"},
+            },
+            "event": {
+                self.PRIMARY_KEYS: {"SendID", "EventType", "SubscriberKey", "EventDate"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"EventDate"},
+            },
+            "folder":{
+                self.PRIMARY_KEYS: {"ID"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"ModifiedDate"},
+            },
+            "list":{
+                self.PRIMARY_KEYS: {"ID"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"ModifiedDate"},
+            },
+            "list_send":{
+                self.PRIMARY_KEYS: {"ListID", "SendID"},
+                self.REPLICATION_METHOD: self.FULL_TABLE,
+            },
+            "list_subscriber":{
+                self.PRIMARY_KEYS: {"SubscriberKey", "ListID"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"ModifiedDate"},
```

Comment:
@harshpatel4crest Are we getting ModifiedDate in the list_subscriber stream?

Reply:
Yes, we are getting it.